cmd/cmount/mount.go

```diff
@@ -2,7 +2,7 @@
 //
 // This uses the cgo based cgofuse library
 
-//go:build cmount && ((linux && cgo) || (darwin && cgo) || (freebsd && cgo) || windows)
+//go:build cmount && ((linux && cgo) || (darwin && cgo) || (freebsd && cgo) || (openbsd && cgo) || windows)
```

Comment:
You need to make the old style and new style build lines match while we need to still support older go version.

Reply:
Hello,

I messed with github and thought draft PRs didn't show up in the main project's wall. Sorry for the noise.

I am not familiar with github or its testing frameworks. I am a bit of a CVS guy.

If you want me to do any specific test I will do it manually in a local computer. We warned I am undergoing a bit of an enforced holiday right now so Internet access is scarce right now.

I will research the build failure.